modules/otel-collector-components.adoc

```diff
@@ -506,13 +596,13 @@ rules:
 ----
 
 [id="filter-processor_{context}"]
-=== Filter processor
+=== Filter Processor
 
-The Filter processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+The Filter Processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
 
-The Filter processor leverages the OpenTelemetry Transformation Language to establish criteria for discarding telemetry data. If any of these conditions are satisfied, the telemetry data are discarded. The conditions can be combined by using the logical OR operator. This processor supports traces, metrics, and logs.
+The Filter Processor leverages the OpenTelemetry Transformation Language to establish criteria for discarding telemetry data. If any of these conditions are satisfied, the telemetry data are discarded. The conditions can be combined by using the logical OR operator. This processor supports traces, metrics, and logs.
```

Comment:
```suggestion
The Filter Processor leverages the OpenTelemetry Transformation Language to establish criteria for discarding telemetry data. If any of these conditions are satisfied, the telemetry data is discarded. You can combine the conditions by using the logical OR operator. This processor supports traces, metrics, and logs.
```

Reply:
Good to have ..

Reply:
I'd like to argue that "telemetry data" is plural. Rewriting the passive voice.